lms/static/js/commerce/views/receipt_view.js

```diff
@@ -35,12 +49,23 @@ var edx = edx || {};
                 },
                 providerId;
 
+            var requires_verification = false;
+
+            var _this = this;
+
+            _.each(data.lines, function (line) {
+               if (_this.requiresVerification(line.product)){
+                   requires_verification = true;
+               }
+            });
```

Comment:
Nit: you could rewrite that as:

```
var requires_verification = _.any(data.lines, function(line) {
    return this.requiresVerification(line.product);
}, this);
```

or if you change `requiresVerification` to take `line` instead of `product`:

```
var requires_verification = _.any(data.lines, this.requiresVerification);
```